src/ProjectSystem/Test/Microsoft.VisualStudio.ProjectSystem.FileSystemMirroring.Test.csproj

```diff
@@ -33,6 +33,13 @@
     <IntermediateOutputPath>$(BaseIntermediateOutputPath)\$(Configuration)\$(AssemblyName)\</IntermediateOutputPath>
     <OutputPath>$(BaseOutputPath)\$(Configuration)\</OutputPath>
   </PropertyGroup>
+  <ItemGroup Condition="'$(VisualStudioVersion)'=='14.0'" >
```

Comment:
Primarily to make sure we reference proper version since installed VS may or may not be appropriate (i.e. we may have Update N but target Update N-x for customers.